internal/pgbackrest/pgbackrest.go

```diff
@@ -60,22 +62,44 @@ func (exec Executor) StanzaCreateOrUpgrade(ctx context.Context, configHash strin
 		stanzaCmd = "upgrade"
 	}
 
+	var reposWithVolumes []v1beta1.PGBackRestRepo
+	for _, repo := range postgresCluster.Spec.Backups.PGBackRest.Repos {
+		if repo.Volume != nil {
+			reposWithVolumes = append(reposWithVolumes, repo)
+		}
+	}
+
+	grep := "grep %s-path /etc/pgbackrest/conf.d/pgbackrest_instance.conf"
```

Comment:
To the point @ValClarkson made, this will handle repos added, but not repos removed.